index.js

```diff
@@ -118,6 +118,10 @@ module.exports = {
             },
         },
         data: {
+            DataWrapper: require('./lib/storage/data/DataWrapper'),
+            MultipleBackendGateway:
+            require('./lib/storage/data/MultipleBackendGateway'),
+            parseLC: require('./lib/storage/data/LocationConstraintParser'),
```

Comment:
The convention here seems to be using Pascal case for exporting files with classes of the same name. I'd expect LocationConstraintParser.js to have a class `LocationConstraintParser`, instead it exports a helper method `parseLC` which is a bit confusing.

Reply:
Yeah, I debated about that, but it would have required a lot of modification of the code both here and in CloudServer, so I thought it wouldn't be worth all the work just to maintain the naming convention.

Reply:
I guess my suggestion in that case would be to rename the file to parseLC.js.